views/01UCS_LAL-PRIMO_REDESIGN/css/ucla/account.css

```diff
@@ -67,3 +67,30 @@ md-list-item p.text-headings {
   color: var(--color-black);
   font-size: 20px !important;
 }
+
+/* First column of fine information (labels) */
+prm-fines
+  md-list
+  md-list-item
+  .md-list-item-text
+  .weak-text
+  > p
+  > span:first-child {
```

Comment:
@kjallen The spec I see for the label (like FINE TYPE and FINE DATE) is 
```
color: var(--black, #0F0F0F);

/* U/Body/Overline */
font-family: Karbon;
font-size: 16px;
font-style: normal;
font-weight: 700;
line-height: 160%; /* 25.6px */
letter-spacing: 1.6px;
text-transform: uppercase;
```

@ztucker4 and I have been pasting in the full Typography spec.  I see some of it here, but not all.

Reply:
Corrected to reflect full typography value